scala/scala-impl/test/org/jetbrains/plugins/scala/annotator/ForComprehensionHighlightingTest.scala

```diff
@@ -506,4 +505,96 @@ class ForComprehensionSemicolonTest extends ForComprehensionHighlightingTestBase
 
   def test_error_semicolons_with_newlines(): Unit =
     assertEquals(2 + 2 + 3 + 3, errors("for{\n; \n; \nx <- Seq(1)\n;\n;\n;\nif x == 3\n;\n;;;\n y = x\n;;\n;} ()"))
-}
\ No newline at end of file
+}
+
+class ForComprehensionRefutableTest_3 extends ForComprehensionHighlightingTestBase {
+
+  import Message.Error
+
+  override protected def supportedIn(version: ScalaVersion): Boolean = version >= LatestScalaVersions.Scala_3_0
```

Comment:
Let's use the exact version `Scala_3_3`

Reply:
Do you mean only to bump to 3.3 or change it to `version == LatestScalaVersions.Scala_3_3`? 
If the latter, why? These tests should pass for all Scala 3 versions (as of right now at least).